about.html

```diff
@@ -0,0 +1,130 @@
+                <nav>
+                    <a href="index.html"><img src="./images/logo.png" class="nav-img" alt="logo"></a>
+                </nav>
+                <ul class="web-nav-ul">
+                    <li class="web-nav-list">
+                        <a href="index.html" class="web-nav-link">Home</a>
+                    </li>
+                    <li class="web-nav-list">
+                        <a href="#program" class="web-nav-link">Program</a>
+                    </li>
+                    <li class="web-nav-list">
+                        <a href="#devs" class="web-nav-link">Developers</a>
+                    </li>
+                    <li class="web-nav-list">
+                        <a href="#partner" class="web-nav-link">Sponsor</a>
+                    </li>
+                    <li class="web-nav-list">
+                        <a href="#" class="web-nav-link active">CN Campaign</a>
+                    </li>
+                </ul>
+            </nav>
+        </div>
+        <div id="myNav" class="overlay">
+            <a class="closebtn"> &times; </a>
+            <div class="overlay-content">
+                <a href="index.html" class="web-nav-link">Home</a>
+                <a href="#program" class="web-nav-link">Program</a>
+                <a href="#devs" class="web-nav-link">Developers</a>
+                <a href="#partner" class="web-nav-link">Sponsor</a>
+                <a href="#" class="web-nav-link ">CN Campaign</a>
+            </div>
+        </div>
+    </header>
+    <br><br><br>
+    <section class="content">
+        <h1 class="about-title">
+            Microverse Developers Global Conference 2022
+        </h1>
+        <p class="about-description">
+            The Microverse Developers Global Conference brings together
+            the community of Software Developers all over the world. This year,
+            we bring you some of the best Developers from around the world.
+        </p>
+        <p class="about-mail">
+            Please contact us for any further questions about Microverse Developers Global Conference 2022!
+        </p>
+        <h5 class="about-venue">
+            MicroverseDGC2022@yahoo.com
+        </h5>
+    </section>
+
+    <section class="logo">
+        <h1 class="logo-title">Microverse Developers Global Conference 2022 logo</h1>
+        <hr>
+        <p>
+            The Microverse Developers Global Conference 2022 logo was decided
+            through the logo competition from March 14th to March 15th 2022
+        </p>
+        <div class="cn-logo">
+            <img src="images/logo2.png" alt="logo">
+        </div>
+
+    </section>
+
+    <section class="past">
+        <h1 class="past-title">Microverse Developers Global Conference 2022 logo</h1>
+        <hr>
+        <p>
+            Take a look at the past Microverse Developers Global Conference 2022 which took place in Nigeria and in
+            South Africa.
+        </p>
+        <div class="event">
```

Comment:
Implemented but it could be better, the color could go a little deeper while maintaining its transparency.
![Screenshot from 2022-04-12 19-40-01](https://user-images.githubusercontent.com/58692590/163031202-2320c96f-8f70-40f0-8db2-5a470c159df2.png)